adi/adf4350.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) 2022-2025 Analog Devices, Inc.
+#
+# SPDX short identifier: ADIBSD
+
+from adi.attribute import attribute
+from adi.context_manager import context_manager
+
+
+class adf4350(attribute, context_manager):
+    """ADF4350 Microwave Wideband Synthesizer with Integrated VCO
+
+    parameters:
+        uri: type=string
+            URI of IIO context with ADF4350
+    """
+
+    _device_name = ""
+
+    def __init__(self, uri="", device_name="adf4350"):
+        context_manager.__init__(self, uri, self._device_name)
+
+        # Find the device
+        for device in self._ctx.devices:
+            if device.name == device_name:
+                self._ctrl = device
+                self._rxadc = device
+                break
+
+        # Raise an exception if the device isn't found
+        if not self._ctrl:
+            raise Exception("ADF4350 device not found")
+
+    @property
+    def frequency_altvolt0(self):
```

Comment:
Remove the channel suffix here (_altvolt0) to all the related properties as it doesn’t provide much value